pandas/core/generic.py

```diff
@@ -10460,7 +10460,8 @@ def _add_series_or_dataframe_operations(cls):
         Add the series or dataframe only operations to the cls; evaluate
         the doc strings again.
         """
-        from pandas.core.window import EWM, Expanding, Rolling, Window
+        from pandas.core.window import exponentialmoving as em
```

Comment:
use python naming conventions, e.g. ExponentialMoving, and don't abbreviate as this is a class

Reply:
k thanks i'll keep that in mind